### PR TITLE
Nats controller helm chart

### PR DESCRIPTION
**Pull request by montaro**
NATS Controller Helm Chart

### CODE REVIEW

File: resources/eventing/charts/nats-controller/values.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  name: ""
+  annotations: ""
+
+NATS_URL: "nats.nats.svc.cluster.local"
+
+image:
+  # name is the name of the container image for the nats-eventing-controller
+  name: "eventing-controller-nats"
+  # tag is the container tag of the eventing-controller image
+  tag: "PR-10251"
```
Comment (montaro) [inline resources/eventing/charts/nats-controller/values.yaml:15]: To be updated once the component PR #10251 is merged

File: installation/resources/installer-cr-cluster.yaml.tpl [hunk 2]
```diff
@@ -37,8 +37,8 @@ spec:
       namespace: "kyma-system"
     - name: "helm-broker"
       namespace: "kyma-system"
-    # - name: "eventing"
-    #   namespace: kyma-system
+    - name: "eventing"
```
Comment (montaro) [inline installation/resources/installer-cr-cluster.yaml.tpl:40]: @sayanh This should be disabled before the merge, no?

File: resources/eventing/charts/nats-controller/templates/clusterrole.yaml [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+  labels: {{- include "nats-controller.labels" . | nindent 4 }}
+rules:
+{{/*- apiGroups:*/}}
+{{/*  - ""*/}}
+{{/*  resources:*/}}
+{{/*  - services*/}}
+{{/*  verbs:*/}}
+{{/*  - list*/}}
+{{/*  - get*/}}
+{{/*  - watch*/}}
```
Comment (montaro) [inline resources/eventing/charts/nats-controller/templates/clusterrole.yaml:15]: Can we delete this please

**Comment by montaro** [comment]
@sayanh Should we rename the helm chart `eventing/controller` to `beb-controller`?

**Review by sayanh — COMMENTED** [review]
> https://github.com/kyma-project/kyma/pull/10286#issuecomment-756688998

Let's discuss and create a separate issue.

File: resources/eventing/README.md [hunk 4]
```diff
@@ -5,14 +5,24 @@ This Helm chart contains all components required for eventing in Kyma:
...
+- nats-server
 
 ## Event Publisher Proxy
 
 This component receives legacy and Cloud Event publishing requests from the cluster workloads (microservice or Serverless functions) and redirects them to the Enterprise Messaging Service Cloud Event Gateway. It also fetches a list of subscriptions for a connected application. Click [here](../../components/event-publisher-proxy) for more details.
 
 ## Controller
 
-This component manages the internal infrastructure in order to receive an event for all subscriptions.
+This component manages the internal infrastructure in order to receive an event for all subscriptions handled by SAP BEB.
```
Comment (kazydek) [inline resources/eventing/README.md:17]:
```suggestion
This component manages the internal infrastructure in order to receive an event for all subscriptions handled by Business Event Bus (BEB).
```

**Review by sayanh — APPROVED** [review]

**Review by kazydek — APPROVED** [review]

**Pull request merged**